_includes/footer-content.html

```diff
@@ -4,6 +4,7 @@
       <p>About</p>
       <ul class="list-unstyled">
         <li><a href="https://github.com/bazelbuild/bazel/wiki/Bazel-Users">Who's using Bazel</a></li>
+        <li><a href="{{ site.main_site_url }}/support.html">Get Support</a></li>
```

Comment:
We already have a support section in the footer. And the page is already linked (see Support Policy)